lib/luaxtable.c

```diff
@@ -209,11 +327,19 @@ LUNATIK_NEWLIB(xtable, luaxtable_lib, &luaxtable_class, luanetfilter_flags);
 
 static int __init luaxtable_init(void)
 {
+	if ((luaxtable_hooks.match = luarcu_newtable(LUARCU_DEFAULT_SIZE, false)) == NULL)
```

Comment:
you can move this to a macro and use an or here to avoid repeating this pattern..

Reply:
yeah.. you're right, I missed that.. sorry.. but I would move this to a macro anyway.. something like..
```
#define luaxtable_inithook(hook)	(luaxtable_hooks.##hook = luarcu_newtable(LUARCU_DEFAULT_SIZE, false))
```